src/domains/sign/env/prod/terraform.tfvars

```diff
@@ -11,10 +11,17 @@ tags = {
   CostCenter  = "BD100 - STRATEGIC INNOVATION"
 }
 
-storage = {
-  enable_versioning = false
-  delete_after_days = 90
-  replication_type  = "GZRS"
+subnets_cirds = {
+  issuer   = ["10.0.102.0/24"]
```

Comment:
is there any reason on why these starts from 102? in case I'd add a comment with some reference.

Reply:
The reason is that we are not "starting"; instead we are using the `io-commons` VNET and the latter has subnets used by multiple different domains. By looking the VNET address allocation via the console, first available subnet was `102.0/24`.

Please notice that currently there is not any easy way to get an available subnet CIDR

Reply:
can we add a comment on how this value is computed?

Reply:
Sure:

1. Open the Azure console
2. Open the `PROD-IO` subscription and then the `io-p-rg-common` resource group in it
3. Look up for the `io-p-vnet-common` VNET, open it and notice that its CIDR is `10.0.0.0/16`
4. Open the `Subnets` form the left sidebar; you'll see the list of currently allocated subnets.
5. Loop through the subnet lists and look for a free CIDR that will both not overlap with the other subnets and fit within the CIDR of the VNET

A tool link https://www.fryguy.net/wp-content/tools/subnets.html might be helpful in dealing with CIDRs and subnet masks

Reply:
maybe we can add a comment with the az command line needed to get the list

Reply:
Sure, it's

```
az network vnet subnet list --subscription PROD-IO --vnet-name io-p-vnet-common --resource-group io-p-rg-common --output table
```

Reply:
nice! I'd put this as a comment inside the sources for newcomers ;-)